packages/app-storage/src/Query.tsx

```diff
@@ -93,17 +93,42 @@ class Query extends React.PureComponent<Props, State> {
         >
           <Component />
         </Labelled>
-        <Labelled className='storage--actionrow-button'>
-          <Button
-            icon='close'
-            isNegative
-            onClick={this.onRemove}
-          />
+        <Labelled className='.storage--queryrow-button'>
+          {this.renderButtons()}
         </Labelled>
       </div>
     );
   }
 
+  private renderButtons () {
+    const { value: { key: { method, section } } } = this.props;
+
+    const specificFunctionalButtons = [];
+    const defaultButton =
+      <Button
+        icon='close'
+        isNegative
+        onClick={this.onRemove}
+      />;
+    const spreadButton =
+      /* needs an spread content action (wasm byte code)*/
+      <Button
+        text='spread'
+      />;
+
+    const copyButton =
+      /* needs an copy content action (wasm byte code) */
+      <Button
+        text='copy'
+      />;
+
+    if (section === 'substrate' && method === 'code') {
```

Comment:
Due to the completely dynamic nature of Substrate, we would only check for specific sections in exceptional circumstances. Basically what we would rather do is when we render something of type `Bytes`, provide the copy functionality.

This dynamic nature is quite important, if we take a look at https://github.com/polkadot-js/apps/issues/460, we see that a developer is using the base libraries to build something and has a need in this case for copy as well. (In his case, it is also WASM and also quite large)